acbuild/acbuild.go

```diff
@@ -249,7 +249,10 @@ func main() {
 	// Make help just show the usage
 	cmdAcbuild.SetHelpTemplate(`{{.UsageString}}`)
 
-	cmdAcbuild.Execute()
+	err := cmdAcbuild.Execute()
+	if cmdExitCode == 0 && err != nil {
+		os.Exit(3)
```

Comment:
https://www.youtube.com/watch?v=MfB1VdNWtEU

On Fri, Feb 19, 2016 at 2:35 PM, Derek Gonyeo <notifications@github.com>
wrote:

> In acbuild/acbuild.go
> <https://github.com/appc/acbuild/pull/178#discussion_r53460670>:
>
> > @@ -249,7 +249,10 @@ func main() {
> >  	// Make help just show the usage
> >  	cmdAcbuild.SetHelpTemplate(`{{.UsageString}}`)
> >
> > -	cmdAcbuild.Execute()
> > +	err := cmdAcbuild.Execute()
> > +	if cmdExitCode == 0 && err != nil {
> > +		os.Exit(3)
>
> Because I know I used 1 in some places, and couldn't remember if I used 2.
> Why 63 or 105?
> … <#-50058543_>
> On Feb 19, 2016, at 07:59, Jonathan Boulle <notifications@github.com>
> wrote: In acbuild/acbuild.go: > @@ -249,7 +249,10 @@ func main() { > //
> Make help just show the usage >
> cmdAcbuild.SetHelpTemplate(`{{.UsageString}}`) > > - cmdAcbuild.Execute() >
> + err := cmdAcbuild.Execute() > + if cmdExitCode == 0 && err != nil { > +
> os.Exit(3) why 3 and not 63 or 105? ― Reply to this email directly or view
> it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/acbuild/pull/178/files#r53460670>.
>